idea_town/frontend/static-src/app/views/details-view.js

```diff
@@ -0,0 +1,78 @@
+import Model from 'ampersand-model';
+import Collection from 'ampersand-collection';
+
+import BaseView from './base-view';
+import template from '../templates/details-view';
+import ContributorView from './contributor-view';
+
+const Contributors = Collection.extend({
+  model: Model.extend({
+    extraProperties: 'allow'
+  })
+});
+
+export default BaseView.extend({
+  _template: template,
+
+  bindings: {
+    'model.modified_date': {
+      type: (el, value) => new Date(value),
+      hook: 'modified-date'
+    },
+
+    'model.description': {
+      hook: 'modified-date'
+    },
+
+    'model.contribute_url': [{
+      hook: 'contribute-url'
+    },
+    {
+      type: 'attribute',
+      name: 'href',
+      hook: 'contribute-url'
+    }],
+
+    'model.measurements': [{
+      type: 'innerHTML',
+      hook: 'measurements-html'
+    },
+    {
+      type: 'toggle',
+      hook: 'measurements-container'
+    }],
+
+    'model.changelog_url': {
+      type: 'attribute',
+      name: 'href',
+      hook: 'changelog-url'
+    },
+
+    'model.thumbnail': {
+      type: 'attribute',
+      name: 'src',
+      hook: 'thumbnail'
+    },
+
+    'model.details.image': {
```

Comment:
`details` is not just a single object, but [a list of `ExperimentDetails` records in the API](https://github.com/mozilla/idea-town/blob/master/idea_town/experiments/serializers.py#L26). This should be a collection like `Contributors` with an associated view.